app/controllers/defendants_controller.rb

```diff
@@ -23,7 +23,8 @@ def update
       if unlink
         redirect_after_unlink
       else
-        redirect_after_failed_unlink
+        flash.now[:alert] = I18n.t('defendants.unlink.failure', error_messages: error_messages)
+        render 'edit'
       end
     else
       render 'edit'
```

Comment:
https://github.com/ministryofjustice/laa-court-data-ui/pull/241/files#r466398929 ..but for edit

Reply:
Have used a return to achieve this, don't know if good ruby style though?

Reply:
[see comment](https://github.com/ministryofjustice/laa-court-data-ui/pull/241/files#discussion_r466458808)